extensions/amp-sidebar/0.2/amp-sidebar.js

```diff
@@ -443,10 +429,6 @@ export class AmpSidebar extends AMP.BaseElement {
       this.element,
       this.getRealChildren()
     );
-    if (this.historyId_ != -1) {
```

Comment:
Let's discuss / clarify if we do intend to remove history manipulation from sidebar itself period.

Reply:
Per discussion, let's leave it in for now, and you can reference https://github.com/ampproject/amphtml/issues/25080 in a `TODO` to remove or attribute-gate the history manipulation code pending the resolution of that issue.